### PR TITLE
Removed further non-WIT branding. Updated relevant fields and removed unnecessary stuff

### LINKED ISSUE #12: Edit Graduation Range
Remove 2018 as an option (past the 1-year mark) and add 2023

### PR DESCRIPTION
**Pull request by corbinhak**
Fixes #12 , #2

### CODE REVIEW

File: app/client/views/admin/user/user.html [hunk 1]
```diff
@@ -55,10 +55,6 @@
         <input type="text" ng-model="selectedUser.email" />
       </div>
 
-      <div class="field">
```
Comment (uptonm) [inline app/client/views/admin/user/user.html]: we might want to make this a devpost link in the future, but good to remove it for now

**Comment by uptonm** [comment]
I'm going to merge this in so I can quickly change a styling mistake I made a while back lmao

File: app/client/views/admin/user/user.html [hunk 2]
```diff
@@ -149,8 +145,8 @@
       <div class="field"
            ng-hide="isMitStudent">
         <p>
-          Because of limitations imposed by MIT, we are not legally allowed to host
-          non-WIT minors (those under 18) for HackWITus 2015. Checking the box below affirms that you are or will be 18 years or older by September 19th, 2015.
+          Because of limitations imposed by WIT, we are not legally allowed to host
+          non-WIT minors (those under 18) for HackWITus 2019. Checking the box below affirms that you are or will be 18 years or older by September 19th, 2015.
```
Comment (ShawnToubeau) [inline app/client/views/admin/user/user.html]: Change `September 19th, 2015` to `November 9th, 2019`

Comment (corbinhak) [reply app/client/views/admin/user/user.html]: This is the admin side of things, It doesn't really matter.

Comment (ShawnToubeau) [reply app/client/views/admin/user/user.html]: Ah okay

**Pull request merged**